core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadServiceNotification.java

```diff
@@ -153,7 +153,7 @@ public void updateReport(List<DownloadStatus> reportQueue, boolean showAutoDownl
                 iconId = R.drawable.ic_notification_sync_error;
                 intent = ClientConfig.downloadServiceCallbacks.getReportNotificationContentIntent(context);
                 id = R.id.notification_download_report;
-                content = String.format(context.getString(R.string.download_report_content), successfulDownloads, failedDownloads);
+                content = context.getResources().getQuantityString(R.plurals.download_report_content, successfulDownloads, successfulDownloads, failedDownloads);
```

Comment:
The line is a bit too long. That's why the test currently fails. Please break it into two lines.

Reply:
Static code analysis strikes again! Will do :)

Reply:
It's still too long ;) Maximum is 120 characters.

Reply:
Aw dang, I'll sort it out!